platform/android/detect.py

```diff
@@ -119,7 +119,9 @@ def mySpawn(sh, escape, cmd, args, env):
 			env.extra_suffix=".armv7.neon"+env.extra_suffix
 		else:
 			env.extra_suffix=".armv7"+env.extra_suffix
-
+        elif env['android_arch']=='armv8':
```

Comment:
The indentation before your `elif`s lacks the `»` character which represents (in Kate) a tabulation. Which means that your indentation is made of 8 spaces instead of 1 tab, like used for the other lines.